src/content/en/2021/seo.md

```diff
@@ -1,20 +1,723 @@
+
+### Core Web Vitals
+
+Core Web Vitals are the new speed metrics that are part of Google's Page Experience signals. The metrics measure visual load with Largest Contentful Paint (LCP), visual stability with Cumulative Layout Shift (CLS), and interactivity with First Input Delay (FID).
+
+The data comes from the Chrome User Experience Report (CrUX), which records real-world data from opted-in Chrome users.
+
+33% of mobile websites are now passing Core Web Vitals thresholds, up from 20% last year. Most websites are passing FID but website owners seem to be struggling to improve CLS and LCP.
+
+{{ figure_markup(
+   image="core-web-vitals-trend.png",
+   caption="Core web vitals metrics trend.",
+   description="Line chart showing ...",
+   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQ7oSHa6DHYTfZqsuGmIYdlfYVkwrUvOYD_r6soecExV_ZpbbZjmG6watu0hwrOKqK3-inNrt0TfXCO/pubchart?oid=1693723951&format=interactive",
+   sheets_gid="460991760",
+   sql_file="sql.sql"
+   )
+}}
+
+{# TODO SQL file location #}
+
+See the [Performance](./performance) chapter for more on this topic.
+
+## On-Page
+
+Search engines look at your page's content to determine whether it's a relevant result for the search query. Other on-page elements may also impact rankings or appearance on the search engines.
+
+### Metadata
+
+Metadata includes `<title>` elements and `<meta name="description">` tags. Metadata can directly and/or indirectly affect SEO performance. See the next sections for specifics about each tag.
+
+In 2021, 98.77% of desktop and mobile pages had `<title>` elements. 71.1% of desktop and mobile pages had `<meta name="description">` tags.
+
+{{ figure_markup(
+   image="title-meta-description-usage.png",
+   caption="Breakdown of title and meta description usage.",
+   description="Bar chart showing ...",
+   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQ7oSHa6DHYTfZqsuGmIYdlfYVkwrUvOYD_r6soecExV_ZpbbZjmG6watu0hwrOKqK3-inNrt0TfXCO/pubchart?oid=541272297&format=interactive",
+   sheets_gid="1347655296",
+   sql_file="seo-stats.sql"
+   )
+}}
+
+### `<title>` Element
+
+The `<title>` element is an on-page ranking factor that provides a strong hint regarding page relevance and may appear on the SERP. In August 2021 [Google started re-writing more titles in their search results](https://developers.google.com/search/blog/2021/08/update-to-generating-page-titles).
+
+In 2021:
+
+- The median page `<title>` contained 6 words.
+- The median page `<title>` contained 39 and 40 characters on desktop and mobile, respectively.
+- 10% of pages had `<title>` elements containing 12 words.
+- 10% of desktop and mobile pages had `<title>` elements containing 74 and 75 characters, respectively.
+
+Most of these stats are relatively unchanged since last year.
```

Comment:
```suggestion
Most of these stats are relatively unchanged since last year. Reminder that these are titles on homepages, that tend to be shorter than those used on deeper pages.
```